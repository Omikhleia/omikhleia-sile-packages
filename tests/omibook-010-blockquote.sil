\begin[class=omibook]{document}
\script[src=packages/lorem]
\nofolios
\noheaders

Before blockquote.

\begin{blockquote}
\lorem[words=20].

\begin{blockquote}
Blockquote inside blockquote, \lorem[words=20].
\end{blockquote}

\lorem[words=20].
\end{blockquote}

After blockquote.

\end{document}
