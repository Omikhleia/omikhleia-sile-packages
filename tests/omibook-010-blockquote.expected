Set paper size 	595.275597	841.8897729
Begin page
Mx 	72.0276
My 	105.2276
Set font 	Gentium Plus;10;400;;normal;;LTR
T	37 72 73 82 85 72 w=26.8359	(Before)
Mx 	101.5471
T	69 79 82 70 78 84 88 82 87 72 w=45.6055	(blockquote)
Mx 	147.1526
T	17 w=2.2900	(.)
Mx 	91.0276
My 	119.6276
Set font 	Gentium Plus;9.5;400;;normal;;LTR
T	79 82 85 72 80 w=23.1331	(lorem)
Mx 	116.0017
T	76 83 86 88 80 w=23.9402	(ipsum)
Mx 	141.7830
T	71 82 79 82 85 w=20.8323	(dolor)
Mx 	164.4564
T	86 76 87 w=9.5139	(sit)
Mx 	175.8114
T	68 80 72 87 w=19.6494	(amet)
Mx 	197.3019
T	70 82 81 86 72 87 72 87 88 85 w=41.9521	(consetetur)
Mx 	241.0952
T	86 68 71 76 83 86 70 76 81 74 w=40.8481	(sadipscing)
Mx 	283.7844
T	72 79 76 87 85 w=16.5693	(elitr)
Mx 	302.1949
T	86 72 71 w=12.9976	(sed)
Mx 	317.0335
T	71 76 68 80 w=19.5056	(diam)
Mx 	338.3802
T	81 82 81 88 80 92 w=32.6099	(nonumy)
Mx 	372.8312
T	72 76 85 80 82 71 w=28.0732	(eirmod)
Mx 	402.7455
T	87 72 80 83 82 85 w=28.8618	(tempor)
Mx 	433.4485
T	76 81 89 76 71 88 81 87 w=33.5608	(invidunt)
Mx 	468.8504
T	88 87 w=8.3032	(ut)
Mx 	478.9947
T	79 68 69 82 85 72 w=24.6870	(labore)
Mx 	78.5276
My 	131.0276
T	72 87 w=7.6584	(et)
Mx 	88.7299
T	71 82 79 82 85 72 w=25.2205	(dolore)
Mx 	116.4942
T	80 68 74 81 68 w=26.2271	(magna)
Mx 	145.2651
T	68 79 76 84 88 92 68 80 w=36.1121	(aliquyam)
Mx 	181.3772
T	17 w=2.1755	(.)
Mx 	109.0276
My 	144.8276
Set font 	Gentium Plus;9;400;;normal;;LTR
T	37 79 82 70 78 84 88 82 87 72 w=41.4229	(Blockquote)
Mx 	152.2048
T	76 81 86 76 71 72 w=22.1572	(inside)
Mx 	176.1165
T	69 79 82 70 78 84 88 82 87 72 w=41.0449	(blockquote)
Mx 	217.1614
T	15 w=2.0610	(,)
Mx 	220.9769
T	79 82 85 72 80 w=21.9155	(lorem)
Mx 	244.6468
T	76 83 86 88 80 w=22.6802	(ipsum)
Mx 	269.0814
T	71 82 79 82 85 w=19.7358	(dolor)
Mx 	290.5717
T	86 76 87 w=9.0132	(sit)
Mx 	301.3393
T	68 80 72 87 w=18.6152	(amet)
Mx 	321.7089
T	70 82 81 86 72 87 72 87 88 85 w=39.7441	(consetetur)
Mx 	363.2075
T	86 68 71 76 83 86 70 76 81 74 w=38.6982	(sadipscing)
Mx 	403.6601
T	72 79 76 87 85 w=15.6973	(elitr)
Mx 	421.1118
T	86 72 71 w=12.3135	(sed)
Mx 	435.1797
T	71 76 68 80 w=18.4790	(diam)
Mx 	455.4131
T	81 82 81 88 80 92 w=30.8936	(nonumy)
Mx 	96.5276
My 	155.6276
T	72 76 85 80 82 71 w=26.5957	(eirmod)
Mx 	125.5175
T	87 72 80 83 82 85 w=27.3428	(tempor)
Mx 	155.2544
T	76 81 89 76 71 88 81 87 w=31.7944	(invidunt)
Mx 	189.4430
T	88 87 w=7.8662	(ut)
Mx 	199.7034
T	79 68 69 82 85 72 w=23.3877	(labore)
Mx 	225.4853
T	72 87 w=7.2554	(et)
Mx 	235.1349
T	71 82 79 82 85 72 w=23.8931	(dolore)
Mx 	261.4221
T	80 68 74 81 68 w=24.8467	(magna)
Mx 	288.6630
T	68 79 76 84 88 92 68 80 w=34.2114	(aliquyam)
Mx 	322.8744
T	17 w=2.0610	(.)
Mx 	91.0276
My 	170.0276
Set font 	Gentium Plus;9.5;400;;normal;;LTR
T	79 82 85 72 80 w=23.1331	(lorem)
Mx 	116.0017
T	76 83 86 88 80 w=23.9402	(ipsum)
Mx 	141.7830
T	71 82 79 82 85 w=20.8323	(dolor)
Mx 	164.4564
T	86 76 87 w=9.5139	(sit)
Mx 	175.8114
T	68 80 72 87 w=19.6494	(amet)
Mx 	197.3019
T	70 82 81 86 72 87 72 87 88 85 w=41.9521	(consetetur)
Mx 	241.0952
T	86 68 71 76 83 86 70 76 81 74 w=40.8481	(sadipscing)
Mx 	283.7844
T	72 79 76 87 85 w=16.5693	(elitr)
Mx 	302.1949
T	86 72 71 w=12.9976	(sed)
Mx 	317.0335
T	71 76 68 80 w=19.5056	(diam)
Mx 	338.3802
T	81 82 81 88 80 92 w=32.6099	(nonumy)
Mx 	372.8312
T	72 76 85 80 82 71 w=28.0732	(eirmod)
Mx 	402.7455
T	87 72 80 83 82 85 w=28.8618	(tempor)
Mx 	433.4485
T	76 81 89 76 71 88 81 87 w=33.5608	(invidunt)
Mx 	468.8504
T	88 87 w=8.3032	(ut)
Mx 	478.9947
T	79 68 69 82 85 72 w=24.6870	(labore)
Mx 	78.5276
My 	181.4276
T	72 87 w=7.6584	(et)
Mx 	88.7299
T	71 82 79 82 85 72 w=25.2205	(dolore)
Mx 	116.4942
T	80 68 74 81 68 w=26.2271	(magna)
Mx 	145.2651
T	68 79 76 84 88 92 68 80 w=36.1121	(aliquyam)
Mx 	181.3772
T	17 w=2.1755	(.)
Mx 	72.0276
My 	196.4276
Set font 	Gentium Plus;10;400;;normal;;LTR
T	36 73 87 72 85 w=21.0840	(After)
Mx 	95.7958
T	69 79 82 70 78 84 88 82 87 72 w=45.6055	(blockquote)
Mx 	141.4013
T	17 w=2.2900	(.)
End page
Finish
